src/middlewared/middlewared/sqlalchemy.py

```diff
@@ -117,6 +117,23 @@ def result_processor(self, dialect, coltype):
         return self._result_processor
 
 
+class DateTime(TypeDecorator):
```

Comment:
We're using the DateTime type from sqlalchemy for audit databases. This is changing behavior there and maybe not intentional.

Reply:
The point is why are we changing this to naive datetimes? Isn't there a chance we actually want TZ info?

Reply:
This looks like a rather impactful change with zero description of why we're doing it.

Reply:
I updated the PR body.